docs/source/guide/ml.md

```diff
@@ -122,6 +126,27 @@ You can prompt your model to train in several ways:
 
 In development mode, training logs appear in the web browser console. In production mode, you can find runtime logs in `my_backend/logs/uwsgi.log` and RQ training logs in `my_backend/logs/rq.log` on the server running the ML backend, which might be different from the Label Studio server. To see more detailed logs, start the ML backend server with the `--debug` option. 
 
+### Active learning approach
+
+Active learning approach is training method with model finetuning  oppose to classical approach when you train model in batches.
+
+Check our [Webhooks section](/guide/webhooks.html#What-to-use-Label-Studio-webhooks-for) to identify  which events should trigger your model training. 
+
+### Train method specification
+(Deprecated in version 1.4.1)
+
+Your Machine Learning train method signature:
+
+```python
+def fit(self, completions, workdir=None, **kwargs)
+```
+
+The **completions** param used in classical approach - list of annotations, can be a list or a single task in a list. Annotations format is common for Label Studio, check annotations key in task format in [guide](/guide/export.html#Label-Studio-JSON-format-of-annotated-tasks).
+The **completions** param will be empty for active learning approach. Instead of it use **kwargs** params.
+
+The **event** key in **kwargs** is Webhook event that called the fit method. Machine learning backend is checking these events to train your model: 'ANNOTATION_CREATED', 'ANNOTATION_UPDATED', 'ANNOTATION_DELETED', 'PROJECT_UPDATED'. To override this list set self.TRAIN_EVENTS in your model. Check available events in [webhooks guide.](/guide/webhooks.html#What-to-use-Label-Studio-webhooks-for)
+The **data** key in **kwargs** enriches your event with entities like project, task or annotation. Check [API description](/api#tag/Webhooks/) to understand event params.
```

Comment:
The **completions** param will be empty for the active learning approach. Instead of it use **kwargs** params.

The **event** key in **kwargs** is a Webhook event called the fit method. Machine learning backend is checking these events to train your model: 'ANNOTATION_CREATED', 'ANNOTATION_UPDATED', 'ANNOTATION_DELETED', 'PROJECT_UPDATED'. To override this list set self.TRAIN_EVENTS in your model. Check available events in [webhooks guide.](/guide/webhooks.html#What-to-use-Label-Studio-webhooks-for)
The **data** key in **kwargs** enriches your event with entities like a project, task, or annotation. Check [API description](/api#tag/Webhooks/) to understand event params.

Reply:
Fixed